aea/crypto/cosmos.py

```diff
@@ -117,17 +117,11 @@ def sign_message(self, message: bytes, is_deprecated_mode: bool = False) -> str:
         signature_base64_str = base64.b64encode(signature_compact).decode("utf-8")
         return signature_base64_str
 
-    def sign_transaction(self, transaction: Any) -> Any:
+    @staticmethod
+    def sign_default_transaction(transaction: Any, signed_transaction, base64_pbk) -> Any:
```

Comment:
Please provide the type hints for all args and a complete docstring with all params